src/server/game/Handlers/CharacterHandler.cpp

```diff
@@ -866,6 +866,8 @@ void WorldSession::HandlePlayerLogin(LoginQueryHolder* holder)
 
     pCurrChar->SendInitialPacketsBeforeAddToMap();
 
+    bool sentCinematic = false;
```

Comment:
That's actually a hangover from my "test" code that actually implemented it at this point just to see if my idea would work. Totally right that and the setting to true later is pointless.